htdocs/luci-static/resources/view/homeproxy/client.js

```diff
@@ -816,7 +841,16 @@ return view.extend({
 		so = ss.option(form.Flag, 'source_ip_is_private', _('Private source IP'),
 			_('Match private source IP.'));
 		so.default = so.disabled;
-		so.rmempty = false;
+		so.modalonly = true;
+
+		so = ss.option(form.DynamicList, 'ip_cidr', _('IP CIDR'),
+			_('Address Filter Fields: Match IP CIDR with query response.'));
```

Comment:
```suggestion
			_('Match IP CIDR with query response.'));
```

`Address Filter Fields: ` can be omitted IMO